### PR TITLE
Changed Index, CSS and Image files for Desktop Version - All major changes

### PR DESCRIPTION
**Pull request by dheerajsachdeva**
Changed Index, CSS and Image files for Desktop Version - All major changes

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -25,211 +30,371 @@
...
+                <li></li><a href="#"><img src="./img/M.svg"></a>
+                </li>
+                <li></li><a href="https://github.com/dheerajsachdeva"><img src="./img/github.svg"></a>
+                </li>
+                <li></li><a href="#"><img src="./img/victory.svg"></a>
+                </li>
+
+            </ul>
+        </div>
+    </div>
```
Comment (Ibizugbe) [inline index.html:75]: Nice work implementing the desktop view while retaining the responsiveness of the page👍 but your intro section looks a little bit different from the design and it is a requirement that we stick to the design😉 the curve at the left bottom corner of the intro section is currently missing🤗

Your app
your___curve

the design
design_curve

kindly take note of the highlighted spot and make the required change🤗😉

File: style.css [hunk 2]
```diff
@@ -671,3 +735,336 @@
...
+  }
+
+  .about-text {
+    width: 447px;
+    height: 102px;
+  }
+
+  .about-myself-upper h2 {
+    padding-top: 45px;
+  }
```
Comment (Ibizugbe) [inline style.css]: Your about section looks a bit different from the design😉 I think we can make some comparison with the image below and add the necessary changes😉 don't forget, it is a requirement that we stick to the design😉

your app
your___about

design
design__about

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @dheerajsachdeva 👋, 

68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f78554f786668365a4d373565664d3342716f2f67697068792d646f776e73697a65642d6d656469756d


Good job so far!👍
There are some issues that you still need to work on to go to the next project but you are almost there!👍

## Highlights✅
- Github flow has been used👍
- Project is properly documented👍

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by frank1738 — APPROVED** [review]
Hi @dheerajsachdeva , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

✅ Correct use of Github workflow
✅ No linter error

## Optional suggestions

N/A



_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**